app/src/main/java/com/jerboa/ui/components/login/Login.kt

```diff
@@ -141,30 +144,35 @@ fun LoginForm(
                 },
                 keyboardOptions = KeyboardOptions(autoCorrect = false, keyboardType = KeyboardType.Uri),
             )
-            ExposedDropdownMenu(
-                expanded = expanded,
-                onDismissRequest = {
-                    expanded = false
-                },
-            ) {
-                instanceOptions.forEach { selectionOption ->
-                    DropdownMenuItem(
-                        modifier = Modifier.exposedDropdownSize(),
-                        text = {
-                            Text(text = selectionOption)
-                        },
-                        onClick = {
-                            instance = selectionOption
-                            expanded = false
-                        },
-                        contentPadding = ExposedDropdownMenuDefaults.ItemContentPadding,
-                    )
+            val filteringOptions = instanceOptions.filter { it.contains(instance, ignoreCase = true) }
```

Comment:
This worked a little strange, IE you have to type some letters, then hit the popup for it to filter. But I also remember the other one I had to revert because it had really terrible performance.